docs/designs/2026-01-29-athenahealth-pivot-mvp.md

```
@@ -141,695 +362,486 @@ app.MapGet("/api/events", async (HttpContext ctx, CancellationToken ct) =>
 
-    A->>D: 1. Launch with ?iss=...&launch=...
-    D->>A: 2. GET /.well-known/smart-configuration
-    A-->>D: 3. {authorization_endpoint, token_endpoint}
-    D->>A: 4. Redirect to authorization_endpoint
-    Note over D,A: User authorizes (may be silent if pre-approved)
-    A->>D: 5. Redirect with ?code=...
-    D->>A: 6. POST token_endpoint (exchange code)
-    A-->>D: 7. {access_token, patient, encounter}
-    D->>G: 8. Connect SSE with access_token
-    G-->>D: 9. Real-time notifications
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)